R/conditionalJS.R

```diff
@@ -118,7 +118,7 @@ json_val <- function(value) {
 json_settings <- function(...) {
   args <- list(...)
   json_args <- purrr::imap(args, ~ glue::glue("\"{.y}\": {json_val(.x)}"))
-  glue::glue("{{{paste(json_args, collapse = ', ')}}}")
+  glue::glue("{<paste(json_args, collapse = ', ')>}", .open = "<", .close = ">")
```

Comment:
I'm wonder why we are not using jsonlite here.

Reply:
With jsonlite I couldn't store raw objects e.g. `{"fun": my_fun}` in the configuration. Don't remember the details now, but it was the only option.